cmd/occollector/app/collector/collector_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package collector handles the command-line, configuration, and runs the OC collector.
+package collector
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestApplication_Start(t *testing.T) {
+	// Without exporters the collector will start and just shutdown, no error is expected.
+	App.v.Set("logging-exporter", true)
+	appDone := make(chan struct{})
+	go func() {
+		if err := App.Start(); err != nil {
+			t.Fatalf("App.Start() got %v, want nil", err)
+		}
+		close(appDone)
```

Comment:
Please make this a defer at the very top of the goroutine `defer close(appDone)`